modules/DMZ/models/model_properties.py

```diff
@@ -0,0 +1,10 @@
+# This file will be part of forcing models to have specific properties that govern
+# their usage.
+
+
+class ModelProperties(object):
+
+    def __init__(self, regression=False, online=False):
+        self._classification = not regression  # I don't know if I should get fancy...
```

Comment:
How ugly are we willing to go with this? [StackOverflow has an opinion](http://stackoverflow.com/questions/3711657/can-i-prevent-modifying-an-object-in-python) on what we _could_ do....